packages/client/hmi-client/src/petrinet/petrinet-renderer.ts

```diff
@@ -74,35 +77,52 @@ export class PetrinetRenderer extends BasicRenderer<NodeData, EdgeData> {
 			.attr('height', (d) => d.height)
 			.attr('y', (d) => -d.height * 0.5)
 			.attr('x', (d) => -d.width * 0.5)
+			.attr('rx', '6')
+			.attr('ry', '6')
 			.style('fill', '#FFF')
-			.attr('stroke', '#AAA')
-			.attr('stroke-width', 2);
+			.attr('stroke', 'var(--petri-nodeBorder)')
+			.attr('stroke-width', 1);
+
+		function handleMouseOver() {
+			d3.select(this).transition().duration(100).attr('r', 8);
+		}
+
+		function handleMouseOut() {
+			d3.select(this).transition().duration(100).attr('r', 4);
+		}
 
 		// transitions drag handles
 		const transitionsHandles = [
 			...transitions
 				.append('circle')
-				.attr('cx', (d) => d.width)
-				.attr('r', (d) => d.width * 0.2),
+				.attr('cx', (d) => d.width * 0.75)
+				.attr('r', '4')
+				.on('mouseover', handleMouseOver)
+				.on('mouseout', handleMouseOut),
 			...transitions
 				.append('circle')
-				.attr('cy', (d) => d.height)
-				.attr('r', (d) => d.width * 0.2),
+				.attr('cy', (d) => d.height * 0.75)
+				.attr('r', '4')
+				.on('mouseover', handleMouseOver)
+				.on('mouseout', handleMouseOut),
 			...transitions
 				.append('circle')
-				.attr('cx', (d) => -d.width)
-				.attr('r', (d) => d.width * 0.2),
+				.attr('cx', (d) => -d.width * 0.75)
+				.attr('r', '4')
```

Comment:
Should a a number-type, can we make this a constant instead of magic number/string?

Reply:
I don't know.. it works.. feel free to optimize if you like